srcjs/input_binding_select.js

```diff
@@ -123,15 +123,7 @@ $.extend(selectInputBinding, {
       this.setValue(el, data.value);
     }
 
-    if (data.hasOwnProperty('label')) {
-      let escaped_id = $escape(el.id);
-      if (this._is_selectize(el)) {
-        escaped_id += "-selectized";
-      }
-      $(el).parent().parent()
-        .find('label[for="' + escaped_id + '"]')
-        .text(data.label);
-    }
```

Comment:
@wch any idea why `parent()` is called twice here but only once in the `getState()` method (above)? Maybe it should be only called once?

Reply:
I don't think `getState` has been used for a long time, so I wouldn't use that as a guide of the correct way to do things. Does a single `parent()` with both kinds of `selectInput`s -- those with `selectize=TRUE` and `selectize=FALSE`?

Reply:
Oh, alright, a single parent appears to work for `selectInput()` but not `selectizeInput()`